scripts/Package_BGA/bga.py

```diff
@@ -65,17 +67,17 @@ def crtYdRound(x):
     xChamferFab = xLeftFab + chamfer
     xPadLeft = xCenter - pitch * ((layoutX - 1) / 2.0)
     xPadRight = xCenter + pitch * ((layoutX - 1) / 2.0)
-    xLeftCrtYd = crtYdRound(xCenter - (pkgWidth / 2 + crtYdOffset))
-    xRightCrtYd = crtYdRound(xCenter + (pkgWidth / 2 + crtYdOffset))
+    xLeftCrtYd = crtYdRound(xCenter - (pkgWidth / 2 + float(crtYdOffset)))
+    xRightCrtYd = crtYdRound(xCenter + (pkgWidth / 2 + float(crtYdOffset)))
```

Comment:
This still doesn't do floating-point division (I meant to write `pkgWidth / 2.0` instead of `pkgWidth / 2`).